.github/dependabot.yaml

```diff
@@ -58,6 +58,8 @@ updates:
       interval: "monthly"
       time: "06:00"
       timezone: "America/Chicago"
+    reviewers:
+      - "coder/frontend"
```

Comment:
I think we changed the name to coder/ts 🤔 but just a thing to be verified. Thx, a lot for this PR.

Reply:
can you confirm I could not see the team. there are 7 teams but only 3 are visible.